frontend/src/restInterfaces/AccountRestInterface.ts

```diff
@@ -65,6 +65,16 @@ export class AccountRestInterface extends BaseRestInterface {
   public static async getOwnAccount(): Promise<Partial<IUser>> {
     return await BaseRestInterface.get<Partial<IUser>>(this.account_path);
   }
+  /**
+   * Sends request to backend to get the account of the user with the given id
+   * @param id the id of the user
+   * @returns the requested user
+   */
+  public static async getAccount(id: number): Promise<Partial<IUser>> {
```

Comment:
This method doesn't even have a valid endpoint in the backend - I'd remove it here and implement it with the admin view, once that's done

Reply:
there should be one and that is :`accountRouter.get('/:ids', async (req, res) => { ...` (line 121 in accountrouter), it just didnt exited in the AccountRestInterface. But i dont need it anymore so i can remove it if you want

Reply:
That one has a method in the interface aswell, `getAccounts`, but that endpoint is only for querying usernames to ids, not full user objects

Reply:
kk i'll remove it